device-types/Cisco/WS-C4510R+E.yaml

```diff
@@ -35,6 +27,6 @@ module-bays:
   - name: Slot 10
     position: '10'
   - name: PS1
-    position: '1'
```

Comment:
The issue with changing the module bay position, is that the module variable in the module template will only be expecting a number rather than PS1.